### PR TITLE
[bext-ut] Updated bext-ut to v2.0.1

### PR DESCRIPTION
**Pull request by Beafantles**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/bext-ut/portfile.cmake [hunk 1]
```diff
@@ -1,26 +1,37 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
 vcpkg_from_github(
-    OUT_SOURCE_PATH SOURCE_PATH
-    REPO boost-ext/ut
-    REF bf8388f61103571dee3061a4ef23292a320d9dbf #committed on 2023-07-09
-    SHA512 e7f95c71fb094170e0f431af115845f66c53f05748829a547612ae480839339b7794d4a3d8c2ae44ad2536654228f00d9d6b058b3b55c4af3432936efc2f6c2d
-    HEAD_REF master
-)
+  OUT_SOURCE_PATH
```
Comment (JonLiu1993) [inline ports/bext-ut/portfile.cmake:4]: @Beafantles, thanks for the PR, could you restore the format of portfile.cmake? The previous format was relatively standardized, you only need to add the code you need to change.

File: ports/bext-ut/portfile.cmake [hunk 2]
```diff
@@ -14,13 +14,20 @@ vcpkg_cmake_configure(
         -DBOOST_UT_BUILD_EXAMPLES=OFF
         -DBOOST_UT_BUILD_TESTS=OFF
         -DINCLUDE_INSTALL_DIR=include
+        -DBOOST_UT_DISABLE_MODULE=ON
 )
+
```
Comment (JonLiu1993) [inline ports/bext-ut/portfile.cmake:19]:
```suggestion

```

File: ports/bext-ut/portfile.cmake [hunk 3]
```diff
@@ -14,13 +14,20 @@ vcpkg_cmake_configure(
         -DBOOST_UT_BUILD_EXAMPLES=OFF
         -DBOOST_UT_BUILD_TESTS=OFF
         -DINCLUDE_INSTALL_DIR=include
+        -DBOOST_UT_DISABLE_MODULE=ON
 )
+
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()
+
```
Comment (JonLiu1993) [inline ports/bext-ut/portfile.cmake:22]:
```suggestion

```

File: ports/bext-ut/portfile.cmake [hunk 4]
```diff
@@ -14,13 +14,20 @@ vcpkg_cmake_configure(
...
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug"
                     "${CURRENT_PACKAGES_DIR}/lib"
 )
 
-vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")
+
+configure_file("${CMAKE_CURRENT_LIST_DIR}/usage"
+               "${CURRENT_PACKAGES_DIR}/share/${PORT}/usage" COPYONLY)
+
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")
```
Comment (JonLiu1993) [inline ports/bext-ut/portfile.cmake:33]:
```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")

```

**Comment by JonLiu1993** [comment]
After change please run command `./vcpkg x-add-version bext-u --overwrite-version` and commit again.

**Comment by Beafantles** [comment]
Alright, I believe everything should be good now.
Tell me if anything else is required!

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Tested usage successfully by `bext-ut:x64-windows`:
````
bext-ut provides CMake targets:

find_package(ut CONFIG REQUIRED)
target_link_libraries(main PRIVATE Boost::ut)
````

**Pull request merged**